renovate.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": ["github>redhat-developer/app-services-renovate-config"],
+  "ignoreDeps": ["parcel"],
```

Comment:
We are not using parcel, this can be skipped

Reply:
done